packages/@ember/-internals/glimmer/tests/integration/helpers/element-action-test.js

```diff
@@ -712,6 +712,45 @@ moduleFor(
       this.assert.equal(originalHandlerWasCalled, true, 'the click handler was called');
     }
 
+    ['@test multiple actions with bubbles=false for same event are called but prevent bubbling']() {
+      let clickAction1WasCalled = false;
+      let clickAction2WasCalled = false;
+      let eventHandlerWasCalled = false;
+
+      let ExampleComponent = Component.extend({
+        actions: {
+          clicked1() {
+            clickAction1WasCalled = true;
+          },
+          clicked2() {
+            clickAction2WasCalled = true;
+          },
+        },
+      });
+
+      this.registerComponent('example-component', {
+        ComponentClass: ExampleComponent,
+        template: strip`
+        <a href="#"
+          {{action "clicked1" on="click" bubbles=false}}
+          {{action "clicked2" on="click" bubbles=false}}
+        >click me</a>`,
+        click() {
+          eventHandlerWasCalled = true;
+        },
+      });
+
+      this.render('{{example-component}}');
+
+      runTask(() => {
+        this.$('a').trigger('click');
```

Comment:
A question about this (as @nlfurniss and I are working on upgrading an app): this PR says it is fixing the no-jQuery `eventDispatcher`, but this test *appears* (and perhaps I'm simply misunderstanding this?) to be adding tests here and below for jQuery-based event trigger. Is this using jQuery *only* for event triggering, but *not* for dispatch? Thanks!

Reply:
@chriskrycho You mean because of that `$()` call *looking* like jQuery? 

If so, it is actually *not*  jQuery here. Was also confused when I first saw this. But there is some test infrastructure that adds this without actually using jQuery. Probably helped migrating tests away from jQuery back then (my *guess*). See 
* https://github.com/simonihmig/ember.js/blob/52b6657d1984a2b524d980be7a290de416053bc0/packages/internal-test-helpers/lib/test-cases/abstract.js#L88 and 
* https://github.com/simonihmig/ember.js/blob/52b6657d1984a2b524d980be7a290de416053bc0/packages/internal-test-helpers/lib/node-query.js

Reply:
We thought that might be the case but wanted to double check. Thank you very much for clarifying!